### PR TITLE
Add hardening scripts.

### PR DESCRIPTION
**Pull request by jmcarp**
Part of https://github.com/18F/cg-atlas/issues/62.

A few questions:
- It looks like we've been pinning all releases versions here so far. Should we continue do that with the hardening scripts? That would break rules updates in snort, since the rules are packaged in to the bosh release.
- Some of the docker jobs included collectd, and some didn't. I threw all the docker release templates into `meta.docker_templates`, so that they're now all using collectd--I'll revert if that's not what we want to do.

### CODE REVIEW

**Comment by sharms** [comment]
- Not sure about question 1 can you rephrase?
- Docker hosts should run collectd, docker containers should run a single process (not collectd)

**Comment by jmcarp** [comment]
Sorry, hopefully these questions make more sense:
- All the bosh releases before this patch pinned to a specific version, like `{name: 'collectd', version: 4}`. I'm wondering if we should also pin release versions for the hardening tools.
- Some docker hosts were running collectd (iirc docker-highmem) and some weren't (docker-compute and docker-compute-reserved). Was that intentional? I set all three hosts to run collectd in this patch.

**Comment by rogeruiz** [comment]
@jmcarp @sharms,

I'm going to 
- keep the bosh releases pinned to a specific version instead of `latest`, since this can be another PR later on for `bosh` 18F/cg-atlas#62.
- As far as `collectd` and docker `hosts` versus `containers`, I think that's unrelated to => 18F/cg-atlas#62.

**Comment by jmcarp** [comment]
@rogeruiz: we probably don't want to pin `snort`, since that would prevent the docker vms from getting updated rules. And changing `collectd` wasn't part of the issue--it's just a collateral fix I put in.
